src/widgets/current-refined-values/__tests__/defaultTemplates-test.js

```diff
@@ -0,0 +1,40 @@
+import defaultTemplates from '../defaultTemplates.js';
+
+describe('current-refined-values defaultTemplates', () => {
+  describe('`item` template', () => {
+    it('has a `item` default template', () => {
+      const item = {
+        type: 'disjunction',
+        label: 'Brand',
+        operator: ':',
+        name: 'Samsung',
+        count: 4,
+        cssClasses: {
+          count: 'ais-current-refined-values--count',
+        },
+      };
+      expect(defaultTemplates.item(item)).toContain(
+        '<span class="ais-current-refined-values--count">4</span>'
+      );
+    });
+    it('wraps query refinements with <q>', () => {
+      const item = {
+        type: 'query',
+        label: 'Query',
+        operator: ':',
+        name: 'Samsu',
+      };
+      expect(defaultTemplates.item(item)).toContain('Query :   <q>Samsu</q>');
+    });
+    it('does not show `count` when query refinement', () => {
+      const item = {
+        type: 'query',
+        label: 'Query',
+        operator: ':',
+        name: 'Samsu',
+        count: 22, // should be ignored
+      };
+      expect(defaultTemplates.item(item)).toContain('Query :   <q>Samsu</q>');
```

Comment:
There shouldn't be a space before the colon (`:`). Why is there so many spaces after the colon?

Reply:
it comes from the ${computedExclude} in the returned string `${computedLabel} ${computedExclude} ${computedName} ${computedCount}`. I would change it to a regexp but I didn't think it mattered and regexp get hard to read with so many `<` `>` `/` involved.
We could write it as       `.toMatch(new RegExp("Query +: +<q>Samsu</q>"));`